### DISCUSSION TITLE
DNS

### DISCUSSION THREAD

**Discussion by legendaryfishwastaken** [3kh0/ext-remover#806] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school has blocked everything. Extensions, bookmarklets, proxies, games, sh1mmer and even the Urban Dictionary. I've had enough. When will you be making the extension launcher DNS. Also, how do you make a dns? I'm curious to try experimenting with it.

**Comment by AshtonDavies** [comment, marked as answer]
Anyone who can host a static webpage for me with a plain regular IP and no hosting providers would be in for a treat. I have a DNS up with no static exploit page to route to.

Right now, it blocks Lightspeed and Hapara Highlights and untested other filters from working without doing any exploits, the update channel without doing CAUB, admin policy updates, website advertisements, trackers, and other stuff.

Having a static exploit page could allow me to bring forth an extension launcher DNS that people want so bad, and even potentially bypass blocked bookmarklets with a little trick/exploit.

**Reply by 3kh0** [reply]
I could do this, what do you want on it?

**Reply by AshtonDavies** [reply]
You can start with a simple JavaScript execution page (textarea and run button), and then credit yourself if you wish.

Then I can share the DNS address to you guys after writing that in.

If you need something, you can ask me here or on Discord (Ashtino#6539).

Thanks for supporting! 🙏👍
